canonical-web-and-design/jaas.ai
# Refactor nav to use Vanilla navigation pattern

## Done

Refactor nav to use Vanilla navigation pattern

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8029
- Check that nav is responsive across breakpoints and that search box works as expected

## Details

Fixes #186
